pages/privacy.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: Privacy
+permalink: /privacy/
+order: 20
+---
+
+# Privacy Statement for the Metal³ Project
+
+As Metal3.io and most of the infrastructure of the Metal³ Project are currently hosted by Red Hat Inc., this site falls under the [Red Hat Privacy Policy](https://www.redhat.com/en/about/privacy-policy){:target="_blank"}.  All terms of that privacy policy apply to this site.  Should we change our hosting in the future, this Privacy Policy will be updated.
```

Comment:
The content is hosted on GitHub. Does that have any bearing on this privacy statement?

Reply:
I think this comes at kubevirt.io (where we based this from) because of statistics, etc and makes reference to the domain ownership, but probably is something to get discussed on the mailing list on the thread I started on 'privacy statement'